packages/amplication-server/src/core/billing/ProvisionSubscriptionDto.ts

```diff
@@ -5,6 +5,7 @@ export class ProvisionSubscriptionDto {
   intentionType: "UPGRADE_PLAN" | "DOWNGRADE_PLAN";
   successUrl: string;
   cancelUrl: string;
+  customerId: string;
```

Comment:
I think we dont need this DTO, instead we can use ProvisionSubscriptionInput

Reply:
yes, you're right